unit_tests/CMakeLists.txt

```diff
@@ -5,27 +5,6 @@ set(CMAKE_CXX_STANDARD 17)
 add_executable(test_libswoc
     unit_test_main.cc
 
-    test_BufferWriter.cc
-    test_bw_format.cc
-    test_Errata.cc
-    test_IntrusiveDList.cc
-    test_IntrusiveHashMap.cc
-    test_ip.cc
-    test_Lexicon.cc
-    test_MemSpan.cc
-    test_MemArena.cc
-    test_meta.cc
-    test_TextView.cc
-    test_Scalar.cc
-    test_swoc_file.cc
-
-    ex_bw_format.cc
-    ex_IntrusiveDList.cc
-    ex_ipspace_properties.cc
-    ex_Lexicon.cc
-    ex_MemArena.cc
-    ex_TextView.cc
-
```

Comment:
please ignore: just to speed up the build

Reply:
You need better hardware.